qemu/tests/cfg/blockdev_stream_install.cfg

```diff
@@ -53,7 +53,7 @@
     # original system image crash when hitting a failure
     image_name_image1 = "images/system_image"
     iscsi_direct:
-        lun_image1 = 1
+        lun_image1 = 3
```

Comment:
hi Aihua,why do we need the change? The default lun is 0 for the image1, then we use lun=1 for another installation could not damage the system image, any other concern?

Reply:
sorry for the confusion, @zhencliu, I described the reason in the bug, but not in this patch. Now I update the commit, you can check the reason there.